src/Features/Core/Portable/RemoveUnnecessaryImports/AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer.cs

```diff
@@ -14,7 +14,7 @@
 namespace Microsoft.CodeAnalysis.RemoveUnnecessaryImports
 {
     internal abstract class AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer
-        : DiagnosticAnalyzer, IBuiltInAnalyzer, IInProcessAnalyzer
+        : DiagnosticAnalyzer, IBuiltInAnalyzer
```

Comment:
so, do we know whether these analyzers can actually run in OOP?

Reply:
if these analyzer now can run in OOP, then that would be awesome.

Reply:
I looked through the services it depends on, and they all appear to be defined in the features layer, which would make them available OOP.